concent_api/verifier/tests/test_utils.py

```diff
@@ -1,33 +1,200 @@
+        parsed_files_to_compare = parse_result_files_with_frames(
+            frames=self.frames,
+            result_files_list=self.result_files_list,
+            output_format=self.output_format,
+        )
+
+        self.assertEqual(parsed_files_to_compare, self.parsed_files_to_compare)
+
+    def test_that_parse_results_files_with_frames_function_should_return_empty_dict_because_of_wrong_output_format(self):
+        parsed_files_to_compare = parse_result_files_with_frames(
+            frames=self.frames,
+            result_files_list=self.result_files_list,
+            output_format='JPG',
+        )
+
+        self.assertEqual(parsed_files_to_compare, {})
+
+    def test_that_render_images_by_frames_function_should_return_correct_output_files_names(self):
+        with mock.patch('verifier.utils.render_image', autospec=True) as mock_render_image:
+            (blender_output_file_name_list, parsed_files_to_compare) = render_images_by_frames(
+                parsed_files_to_compare=self.parsed_files_to_compare,
+                frames=self.frames,
+                output_format=self.output_format,
+                scene_file=self.scene_file,
+                subtask_id=self.subtask_id,
+                verification_deadline=None
+            )
+            self.assertEqual(self.correct_parsed_all_files, parsed_files_to_compare)
+            self.assertEqual(self.correct_blender_output_file_name_list, blender_output_file_name_list)
+            self.assertEqual(mock_render_image.call_count, 2)
+
+    def test_that_upload_blender_output_file_should_correctly_upload_files(self):
+        with mock.patch('verifier.utils.try_to_upload_blender_output_file', autospec=True) as mock_try_to_upload:
+            try:
+                upload_blender_output_file(
+                    frames=self.frames,
+                    blender_output_file_name_list=self.correct_blender_output_file_name_list,
+                    output_format=self.output_format,
+                    subtask_id=self.subtask_id,
+                )
+            except Exception:  # pylint: disable=broad-except
+                self.fail()
+
+        self.assertEqual(mock_try_to_upload.call_count, 2)
+
+    def test_that_method_should_raise_verification_mismatch_when_any_result_file_missing(self):
+        with self.assertRaises(VerificationMismatch):
+            ensure_enough_result_files_provided(
+                frames=[1, 2, 3],
+                result_files_list=self.result_files_list,
+                subtask_id=self.subtask_id,
+            )
+
+    def test_that_method_should_accept_frames_and_result_files_when_correct_variable_is_passed(self):
+        try:
+            ensure_enough_result_files_provided(
+                frames=self.frames,
+                result_files_list=self.result_files_list,
+                subtask_id=self.subtask_id,
+            )
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Same here

Reply:
Pylint is right. This handler should be removed. The test framework will catch the exception anyway.

If you do it like this you silence everything, even stuff like `SyntaxError` and we don't want that.